website/content/en/preview/concepts/disruption.md

```diff
@@ -198,7 +198,13 @@ To enable interruption handling, configure the `--interruption-queue` CLI argume
 
 ## Controls
 
-### Disruption Budgets
+### TerminationGracePeriod 
+
+This is the duration of time that a node can be draining before it's forcibly deleted. A node begins draining when it's deleted. Pods with TerminationGracePeriodSeconds will be deleted preemptively before this terminationGracePeriod ends to give as much time to cleanup as possible. Note that if your pod's terminationGracePeriodSeconds is larger than this terminationGracePeriod, Karpenter may forcibly delete the pod before it has its full terminationGracePeriod to cleanup. 
```

Comment:
Only thing I would clarify here is what you mean by preemptively deleted. It'd probably be more clear to specify exactly when we do the deletion and that it is based on the terminationGracePeriodSeconds of the pod that we are trying to drain.